typeform/typeform.py

```diff
@@ -113,12 +120,15 @@ def _request(self, url, params=None):
     def _build_params(self, form, batch_size):
         """ construct the relevant params according to the Typeform API """
         page_size = batch_size if batch_size else BATCH_SIZE
+        completed = FORM_TYPES[self.source.get('__formTypes')]
         params = {
             'page_size': page_size,
             'sort': 'landed_at,desc',
-            'completed': 1,  # remove the line if you want to include both
+            # 'completed': 1,  # remove the line if you want to include both
```

Comment:
No reason to keep this line.

Reply:
I'll change it in the next PR since I don't permissions to commit to David's branch.